components/dialog/ConfirmationDialogContent.tsx

```diff
@@ -1,18 +1,19 @@
 import { DialogClose, DialogContent } from "@components/Dialog";
 import * as DialogPrimitive from "@radix-ui/react-dialog";
 import { ExclamationIcon } from "@heroicons/react/outline";
-import React from "react";
+import React, { PropsWithChildren } from "react";
+import { Button } from "@components/ui/Button";
 
-export default function ConfirmationDialogContent({
-  title,
-  alert,
-  confirmBtnText,
-  cancelBtnText,
-  onConfirm,
-  children,
-}) {
-  confirmBtnText = confirmBtnText || "Confirm";
-  cancelBtnText = cancelBtnText || "Cancel";
+export type ConfirmationDialogContentProps = {
+  confirmBtnText?: string;
+  cancelBtnText?: string;
+  onConfirm: (event: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+  title: string;
+  alert?: string;
```

Comment:
- use literal strings :)
- call it `variety` similar to material UI's [alert](https://material-ui.com/components/alert/)

```suggestion
  /**
   * @default info
   */
  variety?: 'error' | 'info' /* ... */;
```